src/TopBar/index.tsx

```diff
@@ -15,16 +15,28 @@ export interface Props {
   title?: React.ReactNode;
   customLogo?: React.ReactNode;
   onLogoClick?: Function;
+  theme?: TopBarTheme;
+}
+
+export enum TopBarTheme {
```

Comment:
Our general convention recently has been to use unions of strings instead of enums, eg `"default" | "plain"`, with the advantage being that it's slightly easier for a consumer to use while still giving the same amount of type safety.

Reply:
thanks! 🙏🏿 forget that we're exporting types now!